packages/calling-component-bindings/src/participantListSelector.ts

```diff
@@ -18,23 +18,33 @@ import { getIdentifierKind } from '@azure/communication-common';
 const convertRemoteParticipantsToParticipantListParticipants = (
   remoteParticipants: RemoteParticipantState[]
 ): CallParticipantListParticipant[] => {
-  return remoteParticipants.map((participant: RemoteParticipantState) => {
-    const isScreenSharing = Object.values(participant.videoStreams).some(
-      (videoStream) => videoStream.mediaStreamType === 'ScreenSharing' && videoStream.isAvailable
-    );
+  return remoteParticipants
+    .map((participant: RemoteParticipantState) => {
+      const isScreenSharing = Object.values(participant.videoStreams).some(
+        (videoStream) => videoStream.mediaStreamType === 'ScreenSharing' && videoStream.isAvailable
+      );
 
-    return {
-      userId: toFlatCommunicationIdentifier(participant.identifier),
-      displayName: participant.displayName,
-      state: participant.state,
-      isMuted: participant.isMuted,
-      isScreenSharing: isScreenSharing,
-      isSpeaking: participant.isSpeaking,
-      // ACS users can not remove Teams users.
-      // Removing phone numbers or unknown types of users is undefined.
-      isRemovable: getIdentifierKind(participant.identifier).kind == 'communicationUser'
-    };
-  });
+      return {
+        userId: toFlatCommunicationIdentifier(participant.identifier),
+        displayName: participant.displayName,
+        state: participant.state,
+        isMuted: participant.isMuted,
+        isScreenSharing: isScreenSharing,
+        isSpeaking: participant.isSpeaking,
+        // ACS users can not remove Teams users.
+        // Removing phone numbers or unknown types of users is undefined.
+        isRemovable: getIdentifierKind(participant.identifier).kind === 'communicationUser'
+      };
+    })
+    .sort((a, b) => {
+      const nameA = a.displayName?.toLowerCase() || '';
+      const nameB = b.displayName?.toLowerCase() || '';
+      if (nameA < nameB) {
+        return -1;
+      } else if (nameA > nameB) {
+        return 1;
+      } else return 0;
```

Comment:
nit: keep consistent bracketing
```suggestion
      } else {
        return 0;
      }
```

Perhaps there's a prettifier rule we can turn on for this so its not something anyone has to care about..

Reply:
Yes, a prettier rule would be great.